notes/src/components/notes/Notes.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import { Button } from 'reactstrap';
+// import { Container, LGroup, ListGroupItem } from 'reactstrap';
+import './Notes.css';
+// import logo from '../src/assets/D2rdroid2.png';  //move to ./assets folder
+const url = 'http://localhost:5501/notes'
+// const notes = 'http://localhost:5501/notes'
+
+class Notes extends Component {
+  constructor(props) {  //constructor initializes the default state.
+    super(props);
+    this.state = {
+      notes: []
+    }
+  }
+// ☞ e240912c-8d2f-4025-bcd8-dc8f2f72a0c4
+
+//  componentDidMount() {
+//     fetch(url) // Call the fetch function passing the url of the API as a parameter
+//       .then(res => res.json())
+//       .then(notes => this.setState({notes}, () => console.log('Notes fetched...', notes)
+//         ))
+//         .catch(error => {
+//           console.error('Error fetching notes from mLab:', error); 
+//         })
+//   }
+
+callAPI () {
+  fetch(url) // Call the fetch function passing the url of the API as a parameter
+    .then(res => res.json())
+    .then(notes => this.setState({notes}, () => console.log('Notes fetched...', notes)
+      ))
+      .catch(error => {
+        console.error('Error fetching notes from mLab:', error); 
+      })
+}
+
+componentDidMount() {
+  this.callAPI();
+}
+// removeNote(id) {
+//   this.setState({notes: this.state.notes.filter(note => note.id !== id )});
+// }
+
+  // removeNoteTest() {
+  //   console.log('clicked');
+  //   alert('button was clicked');
+  // }
+
+ deleteFunc (req, res) {
+    console.log(req.params.id);
+    Notes
+      .findByIdAndRemove(req.params.id)
+      .then(note => {
+        res.send('The note was deleted')
+      })
+      .catch(err => console.log(err));
+  };
+  
+
+  render() {
+    return (
+      <div>
+        <div className="View-header">
+         <h1>Notes found about Electric Uprights</h1>
```

Comment:
Try to make this more dynamic so that it could pertain to more than just uprights. I'm sure there is something in your database that tells us what it is and you can pass that as a prop to this component.

Reply:
I plan to after I get it all working.  I wanted to simplify for now to reduce the number variables that could contribute to errors.

My design populates it with the name of the active collection being listed.  That's why i'm so frustrated trying to figure out how to access a specific collection within a db.